### ISSUE TITLE
Chronos: Evaluator enhancement

### ISSUE THREAD

**Issue by TheaperDeng** [intel-analytics/BigDL#5331]
#### Backgroud

Evaluation has become an important topic in our engagement with customers. There are many aspects we need to enhance our evaluator.
- Acurracy
  - Quantitative: Currently we have `Evaluator.evaluate` to cover this function.
  - Qualitative: We need to provide a plot API for our users to check the forecasting/anomaly detection quality.
- Performance
  - We need an API to evaluate the time cost of a workflow
- Usability
  - We need to support both torch and tf users in evaluator.
  - We need to make the import line simplier.

#### Methodology
- Acurracy
  - depend on matplotlib (will define the API)
- Performance
  - `Evaluator.get_latency` (detailed design will be shown in PR.)
- Usability
  - There are 2 possible methods 1. Use numpy ndarray to rewrite most of the metrics 2. Use https://www.tensorflow.org/api_docs/python/tf/keras/metrics to support tf backend
  - change to `from bigdl.chronos.metric import Evaluator`.
